src/app/[locale]/specs/permission/en.mdx

```diff
@@ -0,0 +1,347 @@
+
+Each resource type has a defined set of parameters that can attenuate the permission. A permission can be represented in a string format (for direct use as an OAuth scope), or as a JSON object (for use in permission sets). Permission sets are published as public lexicon schemas, and are limited to lexicon resources under the same NSID hierarchy.
+
+## Resource Types
+
+Permissions relate to account resources on PDS instances:
+
+- `repo`: Public Repository (records and collections)
+- `rpc`: Service Authentication (API calls to external services)
+- `blob`: uploaded media files
+- `identity`: DID and handle
+- `account`: hosting status, email address
+
+There is an additional "meta" resource, `include`, which is used to reference a permission set (see Permission Sets section below).
+
+This section describes these resources in more depth, including all the parameters that can constrain the permission. For each resource, one parameter may be marked as "positional", which impacts the string representation syntax (described in a later section).
+
+Resources and permissions are described in the abstract, not tied to specific PDS API endpoints. For example, `rpc` permissions describe the ability to make remote service API calls using inter-service authentication. This includes both proxied PDS requests (using HTTP headers) and fetching service auth tokens using `com.atproto.server.getServiceAuth`.
+
+### `repo`
+
+Write access to records in the account's public repository. Can be limited to specific record types (collections) or actions (eg update vs delete).
+
+Parameters:
+
+- `collection` (string array, required, positional): NSID of record types. Wildcard (`*`) is allowed and gives access to all records. Partial wildcards are not supported (eg, can not use `com.example.*`)
+- `action` (string array, optional): defines the set of record operations allowed. Allowed values are `create`, `update`, `delete`. If not defined, all operations are allowed
+
+Examples:
+
+```
+# full permission (create, update, delete) on the Bluesky profile record type
+repo:app.example.profile
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"]
+}
+
+# same as above, with actions being explicit
+repo:app.example.profile?action=create&action=update&action=delete
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"],
+  "action": ["create", "update", "delete"]
+}
+
+# full permissions on all record types
+repo:*
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["*"]
+}
+
+# delete only permission on all record types
+repo:*?action=delete
```

Comment:
should we add the JSON representation of this so that it matches the other examples?